### ISSUE TITLE
Postgres Db initialization bug

### ISSUE THREAD

**Issue by parithon** [csharpfritz/CoreWiki#287]
TLDR; We need to add entity framework within the **IdentityHostStartup** class instead of **StartupExtensions**.

When trying to setup Postgres on Friday's stream (9/7/2018) it would fail because we ran into an issue configuring the services. The issue turns out to be because we didn't configure the service to add the entity framework properly. It turns out `CoreWiki.Areas.Identity.IdentityHostingStartup.ConfigureDb` is called before `CoreWiki.Data.EntityFramework.StartupExtensions.AddRepositories` but we didn't setup entity framework within the IdentityHostingStartup; we set it up in StartupExtensions.

https://github.com/csharpfritz/CoreWiki/blob/42ec4cdbbf8db5f9f2f2163a163da507f4881f10/CoreWiki.Data/StartupExtensions.cs#L27

https://github.com/csharpfritz/CoreWiki/blob/42ec4cdbbf8db5f9f2f2163a163da507f4881f10/CoreWiki.Data/StartupExtensions.cs#L31

**Comment by parithon** [comment]
n/m
